doc/integrator/extend_application.rst

```diff
@@ -9,8 +9,98 @@ To add an additional component in the project in simple mode we should:
 - add a custom docker-compose service
 - add a custom JavaScript file
 
-Interface in canvas mode
-------------------------
+For that in this tutorial we will:
+
+- Create an new Docker image for the new service
+- Integrate it to the project
+- Create the new interface based on canvas
+- Create a new WebComponent
+- Built it in the config image
+- Add it to the project
+
+
+Create an new Docker image for the new service
+----------------------------------------------
+
+In this chapter we will create a new Pyramid application that use cornice in a Docker image.
+
+We will use the Pyramid Cookiecutter starter, we can use it directly
+(by running ``cookiecutter gh:Pylons/pyramid-cookiecutter-starter``) if you want to do your
+image by your own, but in this tutorial we will get the files directly from the demo.
+For that run the following command (should be adapted for Windows but you will get the sense):
```

Comment:
I don't think we should mention Windows terminals specifically here, we don't do it in the other places.